src/App/App.js

```diff
@@ -3,6 +3,7 @@ import Navbar from '../Navbar/Navbar';
 import Movie from '../Movie/Movie';
 import AllMovies from '../All-Movies/All-Movies';
 import movieData from '../movieData';
+import { getMovie } from '../api-calls/apiCalls'
 
```

Comment:
I am exporting the `getMovies` method as a object, using destructuring here

Reply:
whoa cool!

Reply:
> @mayakappen Yeah! feel free to use the same API file. You can export the function inside the same object, using commas to separate and destructuring in the App.js.
> 
> Thank you!!!!

At least I think this is how it works.